test/e2e/tests/confirmations/transactions/contract-interaction-redesign.spec.ts

```diff
@@ -26,19 +26,18 @@ const { CHAIN_IDS } = require('../../../../../shared/constants/network');
 describe('Confirmation Redesign Contract Interaction Component', function () {
   const smartContract = SMART_CONTRACTS.PIGGYBANK;
 
-  if (!process.env.ENABLE_CONFIRMATION_REDESIGN) {
-    return;
-  }
-
-  describe('Create a deposit transaction', function () {
+  describe('Create a deposit transaction @no-mmi', function () {
```

Comment:
We should avoid the `@no-mmi` tag unless strictly necessary, because mmi uses the redesigned flows as well, and the component tree includes components specific for them.